dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobType.java

```diff
@@ -62,6 +62,7 @@
     EVENT_PROGRAMS_DATA_SYNC( "eventProgramsDataSyncJob", true,
         EventProgramsDataSynchronizationJobParameters.class, null ),
     FILE_RESOURCE_CLEANUP( "fileResourceCleanUpJob", false, null, null ),
+    IMAGE_PROCESSING( "imageProcessingJob", false, null, null ),
```

Comment:
Perhaps call this `IMAGE_RESIZE_PROCESSING` or something a bit more specific just in case we need other types image processing later? I might be wrong.

Reply:
maybe IMAGE_RESIZING_JOB?